File: app-code/frontend/src/app/core/models/request/pedido-form-request.ts
import { ClienteRequest } from "./cliente-request";
import { PersonaRequest } from "./persona-request";
import { RepresentanteRequest } from "./representante-request";

export interface PedidoFormRequest {
  personaRequest?: PersonaRequest;
  clienteRequest?: ClienteRequest;
  representanteRequest?: RepresentanteRequest;
}

File: app-code/backend/src/main/java/com/sanfernando/sanfernando/dtos/requests/PedidoFormRequest.java
package com.sanfernando.sanfernando.dtos.requests;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class PedidoFormRequest {
  private PersonaRequest personaRequest;
  private ClienteRequest clienteRequest;
  private RepresentanteRequest representanteRequest;
}
